### ISSUE TITLE
Bugged Continue/Retry/Quit buttons in FPoSu mode

### ISSUE THREAD

**Issue by arxenix** [McKay42/McOsu#245]
When you are using the FPoSu mod, and mouse over one of the buttons in the escape menu, the button flickers rapidly and you need to attempt clicking it several times to get it to register correctly. Can provide footage of this issue if needed.

**Comment by McKay42** [comment]
Please record a video, and also all of your fposu settings.
Open the console via `SHIFT` + `F1`, then enter this command: `debug_mouse 1`

**Comment by arxenix** [comment]
@McKay42 Sorry for the delay, but I got it recorded.

https://streamable.com/qn1i2

**Comment by arxenix** [comment]
osu.cfg
```
osu_background_fade_during_breaks 1

fullscreen
osu_songbrowser_sortingtype By Date Added
monitor 0
osu_folder D:\Games\osu\
osu_database_enabled 1
osu_scores_legacy_enabled 0
name arx
osu_user_include_relax_and_autopilot_for_stats 0
osu_scores_sort_by_pp 1
osu_pause_on_focus_loss 1
vsync 0
osu_draw_fps 1
fps_unlimited 0
fps_max 420
fullscreen_windowed_borderless 0
osu_resolution_keep_aspect_ratio 0
osu_letterboxing 1
osu_letterboxing_offset_x 0
osu_letterboxing_offset_y 0
osu_skin_mipmaps 0
osu_snaking_sliders 1
osu_slider_shrink 0
osu_force_legacy_slider_renderer 0
osu_options_high_quality_sliders 0
osu_options_slider_quality 0
osu_volume_master 0.3
osu_volume_music 0.5
osu_volume_effects 1
osu_universal_offset 0
osu_number_scale_multiplier 1
osu_hitresult_scale 1
osu_draw_numbers 1
osu_draw_approach_circles 1
osu_ignore_beatmap_sample_volume 0
osu_ignore_beatmap_combo_colors 0
osu_skin_use_skin_hitsounds 1
osu_skin_hd 1
osu_draw_cursor_trail 1
osu_cursor_trail_smooth_force 0
osu_cursor_scale 0.22
osu_automatic_cursor_size 0
osu_slider_use_gradient_image 0
osu_slider_osu_next_style 0
osu_slider_ball_tint_combo_color 1
osu_slider_border_tint_combo_color 0
osu_slider_draw_endcircle 1
osu_slider_alpha_multiplier 1
osu_slider_body_alpha_multiplier 1
osu_slider_body_color_saturation 1
osu_slider_border_size_multiplier 1
mouse_sensitivity 1.57
mouse_raw_input 0
mouse_raw_input_absolute_to_window 0
osu_confine_cursor_windowed 0
osu_confine_cursor_fullscreen 1
osu_disable_mousewheel 0
osu_disable_mousebuttons 0
osu_draw_cursor_ripples 0
win_realtimestylus 0
win_ink_workaround 0
tablet_sensitivity_ignore 0
osu_key_left_click 90
osu_key_right_click 88
osu_key_game_pause 27
osu_key_skip_cutscene 32
osu_key_toggle_scoreboard 9
osu_key_seek_time 16
osu_key_increase_local_offset 107
osu_key_decrease_local_offset 109
osu_key_quick_retry 8
osu_key_quick_save 117
osu_key_quick_load 118
osu_key_save_screenshot 123
osu_key_increase_volume 38
osu_key_decrease_volume 40
osu_key_disable_mouse_buttons 121
osu_key_boss 45
osu_key_mod_easy 81
osu_key_mod_nofail 87
osu_key_mod_halftime 69
osu_key_mod_hardrock 65
osu_key_mod_suddendeath 83
osu_key_mod_doubletime 68
osu_key_mod_hidden 70
osu_key_mod_flashlight 71
osu_key_mod_relax 90
osu_key_mod_autopilot 88
osu_key_mod_spunout 67
osu_key_mod_auto 86
osu_key_mod_scorev2 66
osu_background_dim 0
osu_background_brightness 0.05
osu_background_dont_fade_during_breaks 0
osu_show_approach_circle_on_first_hidden_object 1
osu_mod_suddendeath_restart 0
osu_load_beatmap_background_images 1
osu_draw_beatmap_background_image 0
osu_draw_songbrowser_background_image 1
osu_draw_songbrowser_thumbnails 1
osu_draw_rankingscreen_background_image 1
osu_draw_menu_background 1
osu_draw_songbrowser_menu_background_image 1
osu_note_blocking 1
osu_draw_hud 1
osu_draw_score 1
osu_draw_combo 1
osu_draw_accuracy 1
osu_draw_progressbar 1
osu_draw_hiterrorbar 1
osu_draw_scoreboard 1
osu_draw_inputoverlay 1
osu_draw_scrubbing_timeline 1
osu_hud_hiterrorbar_showmisswindow 0
osu_draw_statistics_pp 0
osu_draw_statistics_misses 0
osu_draw_statistics_sliderbreaks 0
osu_draw_statistics_bpm 0
osu_draw_statistics_ar 0
osu_draw_statistics_cs 0
osu_draw_statistics_od 0
osu_draw_statistics_nps 0
osu_draw_statistics_nd 0
osu_draw_statistics_ur 0
osu_hud_scale 1
osu_hud_score_scale 1
osu_hud_combo_scale 1
osu_hud_accuracy_scale 1
osu_hud_hiterrorbar_scale 1
osu_hud_progressbar_scale 1
osu_hud_scoreboard_scale 1
osu_hud_inputoverlay_scale 1
osu_hud_statistics_scale 1
osu_draw_followpoints 1
osu_draw_scorebarbg 1
osu_draw_playfield_border 1
osu_hud_playfield_border_size 5
osu_mod_hd_slider_fast_fade 0
osu_slider_scorev2 0
osu_mod_fposu 1
fposu_curved 1
fposu_cube 0
fposu_distance 0.5
fposu_fov 103
fposu_mouse_dpi 800
fposu_mouse_cm_360 28.8636
fposu_invert_vertical 0
fposu_invert_horizontal 0
fposu_absolute_mode 0
osu_rich_presence 1
osu_circle_rainbow 0
osu_slider_rainbow 0
osu_circle_number_rainbow 0
osu_slider_break_epilepsy 0
osu_hide_cursor_during_gameplay 0
osu_hitresult_draw_300s 0
osu_skin_mipmaps 0
osu_skin -+YUGEN+SD+-
```

**Comment by McKay42** [comment]
That's caused by your skin trying to abuse oversized menu element images to move them to the right side, I'll see about including a fix in the next update.

In the meantime you could either just keep your cursor horizontally centered and just move it up/down, or temporarily rename the offending skin file elements so they don't get loaded: pause-back.png, pause-continue.png, pause-retry.png

**Comment by arxenix** [comment]
@McKay42 I'm not entirely convinced that's the issue here. Here's some footage after I removed the offending skin elements you listed, and there still seems to be issues with pause menu item selection:

https://streamable.com/f2w12

**Comment by McKay42** [comment]
Hm, I thought I had found the reason, but now I'm not so sure (can't reproduce so it's hard to validate).
Can you pm me on Discord for testing a debug build? - https://discord.gg/3UQnt7X

**Comment by McKay42** [comment]
Problem was that `m_fposu->update();` was called _after_ the main screen vector update loop, and since it directly modifies the mouse position the result is a 1 frame delay and discrepancy causing the jitter effect on the pause buttons.

Fix will be included in version 30.1
